samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.HttpApiV2TracingSample/ApiGatewayHttpApiV2Tracing.cs

```diff
@@ -21,14 +21,12 @@ protected override Task<APIGatewayHttpApiV2ProxyResponse> Handle(APIGatewayHttpA
 
             //If you need to call another system, you need to obtain a traceparent based on the original traceparent
             //received but with the ParentId changed
-            var currentTraceparent = context.AdditionalContext["traceparent"].ToString();
-            var currentTracestate = context.AdditionalContext["tracestate"].ToString();
-            var newTraceContext = TraceContext.MutateParentId(currentTraceparent, currentTracestate);
-
+            var currentTraceContext = (TraceContext)context.AdditionalContext["TraceContext"];
+            var newTraceContext = TraceContext.MutateParentId(currentTraceContext);
```

Comment:
Should this be a class method?

Reply:
Why not use a static method that returns a new tracecontext?